src/Components/FileInput.js/DrowImages.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const DrowImages = ({image, style}) => (
```

Comment:
Please add spaces between curly brackets, if the object or destructor contains only one line.